html/js/banks.js

```diff
@@ -192,6 +193,12 @@ JqueryClass('bankBox', {
             }, 500)
             self.data('savingTimeout', timeout)
         })
+        // Update displayed indexes
```

Comment:
Why is this needed? It is being called inside the save function, which does not seem the right place for it.

Reply:
it's to cover all the cases at once: dragging, adding and removing banks, it would have to be called here https://github.com/moddevices/mod-ui/blob/indexes/html/js/banks.js#L78 (drag, add) and here https://github.com/moddevices/mod-ui/blob/indexes/html/js/banks.js#L90 (remove) instead